plugin/src/cli_utils/localpv/hostpath/mod.rs

```diff
@@ -32,6 +32,10 @@ pub struct CliArgs {
     /// The Output, viz yaml, json.
     #[clap(global = true, default_value = OutputFormat::None.as_ref(), short, long)]
     pub output: OutputFormat,
+
+    /// Path to kubeconfig file.
+    #[clap(skip)]
```

Comment:
This is because it's global?

Reply:
Yes, We just copy from global arg to this. SImilar to how we handle namespace.